doc/source/conf.py

```diff
@@ -133,7 +133,10 @@
     rst_epilog += f.read()
 
 
-linkcheck_ignore = ["https://sphinxdocs.ansys.com/version/*"]
+linkcheck_ignore = [
+    "https://sphinxdocs.ansys.com/version/*",
+    "https://github.com/ansys/ansys-sphinx-theme/*",
+]
```

Comment:
Why are we ignoring this? Is it causing any issues?

Reply:
The link check was failing https://github.com/ansys/ansys-sphinx-theme/actions/runs/11405170405/job/31736081063 . i will investigate further

Reply:
AFAIK we can't check github links, they are not well handled :/ If it ever worked please let me know, I could look into why this evolved.

Reply:
Hey, please review the logs...

It is complaining because your link is ``ansys/ansys_sphinx_theme`` whereas your repo is ``ansys/ansys-sphinx-theme``... Please fix the links internally.

Reply:
We should not be ignoring GitHub links unless it is pointing to a private repository

Reply:
removed the ignore links, thanks all for the suggestions